packages/grafana-data/src/types/accesscontrol.ts

```diff
@@ -0,0 +1,14 @@
+import { KeyValue } from '.';
```

Comment:
@mckn followed your suggestion. But would you have placed it somewhere else?

Reply:
I think is the place for it! Great work